lib/ansible/modules/database/postgresql/postgresql_user.py

```diff
@@ -207,11 +207,11 @@
     password: NULL
 '''
 
+from distutils.version import StrictVersion
 from hashlib import md5
 import itertools
 import re
 
-from distutils.version import StrictVersion
```

Comment:
`hashlib` module belongs to the Python standard library ([2.7](https://docs.python.org/2.7/library/hashlib.html), [3.6.1](https://docs.python.org/3/library/hashlib.html)).